enterprise/web.core/src/org/netbeans/modules/web/wizards/AttrDialog.java

```diff
@@ -91,24 +91,24 @@ private void initComponents() {//GEN-BEGIN:initComponents
 
         jLabel2.setDisplayedMnemonic(java.util.ResourceBundle.getBundle("org/netbeans/modules/web/wizards/Bundle").getString("A11Y_AttrType_mnem").charAt(0));
         jLabel2.setLabelFor(jComboBox1);
-        jLabel2.setText(org.openide.util.NbBundle.getMessage(AttrDialog.class, "LBL_AttrType"));
+        jLabel2.setText(org.openide.util.NbBundle.getMessage(AttrDialog.class, "LBL_AttrType")); // NOI18N
         gridBagConstraints = new java.awt.GridBagConstraints();
         gridBagConstraints.gridx = 0;
         gridBagConstraints.gridy = 1;
         gridBagConstraints.anchor = java.awt.GridBagConstraints.WEST;
         gridBagConstraints.insets = new java.awt.Insets(0, 6, 6, 0);
         add(jLabel2, gridBagConstraints);
-
         gridBagConstraints = new java.awt.GridBagConstraints();
         gridBagConstraints.gridx = 1;
         gridBagConstraints.fill = java.awt.GridBagConstraints.HORIZONTAL;
         gridBagConstraints.weightx = 1.0;
         gridBagConstraints.insets = new java.awt.Insets(6, 6, 6, 0);
         add(jTextField1, gridBagConstraints);
-        jTextField1.getAccessibleContext().setAccessibleDescription(java.util.ResourceBundle.getBundle("org/netbeans/modules/web/wizards/Bundle").getString("LBL_AttrName"));
+        java.util.ResourceBundle bundle = java.util.ResourceBundle.getBundle("org/netbeans/modules/web/wizards/Bundle"); // NOI18N
```

Comment:
How was this code changed? This does not look like changes done with the Form editor, but this is a guarded section, so did you manually change this code outside NetBeans?

Reply:
Actually this code was generated by the NetBeans UI designer. I could reproduce it on master by:
* Open `AttrDialog`
* Select `jComboBox1`
* Switch to `Code` tab in `Properties`
* Set `Type Parameters` to `String`

Reply:
A thank you - I see. Seems the way the UI designer generates code was update. I justed wanted to make sure, as we had cases where people made mass refectorings with tool outside the IDE and changed the guarded blocks, which breaks/gets removed once the code is regenerated.